handler/src/main/java/io/netty/handler/ssl/JdkSslServerContext.java

```diff
@@ -60,11 +61,29 @@ public JdkSslServerContext(File certChainFile, File keyFile) throws SSLException
      * @deprecated use {@link SslContextBuilder}
      */
     @Deprecated
-    public JdkSslServerContext(File certChainFile, File keyFile, String keyPassword) throws SSLException {
+    public JdkSslServerContext(File certChainFile, File keyFile,
+                               String keyPassword) throws SSLException {
         this(certChainFile, keyFile, keyPassword, null, IdentityCipherSuiteFilter.INSTANCE,
                 JdkDefaultApplicationProtocolNegotiator.INSTANCE, 0, 0);
     }
 
+    /**
+     * Creates a new instance.
+     *
+     * @param certChainFile an X.509 certificate chain file in PEM format
+     * @param keyFile a PKCS#8 private key file in PEM format
+     * @param keyPassword the password of the {@code keyFile}.
+     *                    {@code null} if it's not password-protected.
+     * @param keyStore the used keystore type
+     * @deprecated use {@link SslContextBuilder}
+     */
+    @Deprecated
+    public JdkSslServerContext(File certChainFile, File keyFile,
```

Comment:
It makes no sense to add a new public method just to mark it `@deprecated`. I think you want to make it package-private and remove the `@Deprecated`